src/com/owncloud/android/ui/dialog/WebViewDialog.java

```diff
@@ -213,16 +211,6 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
     }
 
     @Override
-    public void onSaveInstanceState(Bundle outState) {
```

Comment:
Do not we need onSaveInstanceState any more? Are we handling the screen rotation from other point?